src/coreComponents/constitutive/solid/PorousSolid.hpp

```diff
@@ -145,11 +145,13 @@ class PorousSolidUpdates : public CoupledSolidUpdates< SOLID_TYPE, BiotPorosity,
                         dTotalStress_dTemperature, // To pass something here
                         stiffness );
 
-    // Compute effective stress increment for the porosity
+    // Compute total stress increment for the porosity update
     GEOS_UNUSED_VAR( pressure_n, temperature_n );
     real64 const bulkModulus = m_solidUpdate.getBulkModulus( k );
     real64 const meanEffectiveStressIncrement = bulkModulus * ( strainIncrement[0] + strainIncrement[1] + strainIncrement[2] );
-    m_porosityUpdate.updateMeanEffectiveStressIncrement( k, q, meanEffectiveStressIncrement );
+    real64 const biotCoefficient = m_porosityUpdate.getBiotCoefficient( k );
+    real64 const meanTotalStressIncrement = meanEffectiveStressIncrement - biotCoefficient * ( pressure - pressure_n );
```

Comment:
@sohailwaziri @paveltomin I believe there should be a temperature - temperature_n here in the total stress increment computation